components/brave_wallet/browser/asset_ratio_response_parser.cc

```diff
@@ -301,4 +301,134 @@ mojom::BlockchainTokenPtr ParseTokenInfo(const std::string& json,
       *symbol, decimals, true, "", "", chain_id, coin);
 }
 
+bool ParseCoinMarkets(const std::string& json,
+                      std::vector<mojom::CoinMarketPtr>* values) {
+  DCHECK(values);
+  // {
+  //   "payload": [
+  //     {
+  //       "id": "bitcoin",
+  //       "symbol": "btc",
+  //       "name": "Bitcoin",
+  //       "image":
+  //       "https://assets.coingecko.com/coins/images/1/large/bitcoin.png?1547033579",
+  //       "market_cap": 727960800075,
+  //       "market_cap_rank": 1,
+  //       "current_price": 38357,
+  //       "price_change_24h": -1229.64683216549,
+  //       "price_change_percentage_24h": -3.10625,
+  //       "total_volume": 17160995925
+  //     },
+  //     {
+  //       "id": "ethereum",
+  //       "symbol": "eth",
+  //       "name": "Ethereum",
+  //       "image":
+  //       "https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880",
```

Comment:
nit: Should we update these links to display the proxy URL now?

Reply:
@thypon This is a comment showing an example of  a response from coingecko.